python/scripts/bump.py

```diff
@@ -12,6 +12,8 @@
 
 from semver import VersionInfo
 
+print("This is a TEST PR")
```

Comment:
This debug print statement should be removed as it:
1. Breaks the logical flow of the code by being placed between imports and class definitions
2. Adds unnecessary output that could interfere with automated processes
3. Provides no functional value to the codebase